common/logger/src/struct_log.rs

```diff
@@ -38,30 +38,40 @@ const INITIALIZED: usize = 2;
 // severity level - lower is worse
 const SEVERITY_CRITICAL: usize = 1;
 const SEVERITY_WARNING: usize = 2;
+const SEVERITY_SECURITY: usize = 3;
 
 #[derive(Default, Serialize)]
 pub struct StructuredLogEntry {
+    /// log message
     #[serde(skip_serializing_if = "Option::is_none")]
     log: Option<String>,
+    /// description of the log
```

Comment:
I'm going to need help for some of these fields that I'm not sure I understand =)

Reply:
This is a pattern used in info! and similar macro. This is not field otherwise